charts/dvpe-deployment-gloo/templates/certificate.yaml

```diff
@@ -14,40 +15,47 @@ spec:
   commonName: {{$firstDomain}}
   secretName: {{$serviceName}}-private-tls-by-issuer
   issuerRef:
-    {{- with .Values.certificate.customIssuerSel }}
-    name: {{ .name }}
     group: controller-manager.dvpe-cloud.github.io
-    kind: {{ .kind }}
+    {{- if eq .customIssuer "intranet" }}
+    name: wadtfy-intranet-cluster-issuer
+    kind: ClusterIssuer
+    {{- else if eq .customIssuer "internet" }}
+    name: wadtfy-internet-cluster-issuer
+    kind: ClusterIssuer
+    {{- else }}
+    name: .clusterIssuerRef.name
+    kind: .clusterIssuerRef.kind
```

Comment:
In this case the kind can only be of type Issuer since a tenant can not create its own cluster resources. The only possibility left here is to create a custom namespaced issuer. You can hard code the kind in this case and set it to 

kind: Issuer